utils/_context/header_tag_vars.py

```diff
@@ -0,0 +1,51 @@
+prefix = "http.request.headers."
+
+
+class TestData:
+    def __init__(self, header, value, config, tag):
+        self.header = header
+        self.value = value
+        self.config = config
+        self.tag = tag
+
+
+header_tag_tests = {
```

Comment:
From my experience, moving data away from test classe makes tests hard to debug, could you repopulate every data where they are used?